ArrayQuery/testsrc/ArrayQueryTest.java

```diff
@@ -0,0 +1,98 @@
+import static com.mongodb.client.model.Filters.and;
+import static com.mongodb.client.model.Filters.elemMatch;
+import static com.mongodb.client.model.Filters.eq;
+import static com.mongodb.client.model.Filters.gt;
+import static org.junit.Assert.*;
+
+import org.bson.Document;
+
+import java.text.ParseException;
+import java.util.List;
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.MongoDatabase;
+
+import org.junit.Test;
+
+
+public class ArrayQueryTest {
+
+	String CONNECTION = "mongodb+srv://m001-student:student123#@sandbox-trhqa.mongodb.net/test";
+	
+	@Test
+	public void TestConnection() {		
+		MongoClientURI clientUri = new MongoClientURI(CONNECTION);
+		try(MongoClient client = new MongoClient(clientUri))
+		{
+			assertNotNull(client);
+		}    
+	}
+	@Test
+	public void InOperatorTest() {
```

Comment:
This test could be considerably more robust and really should be. You will want to make sure that it works (and does not throw an exception) for corner cases such as no matching results or an empty array passed as a part of the filter. Are there other corner cases we should consider?

Also, it would be ideal to test that the documents returned are the ones you expect rather than just count the results. This ensures that if our writeup of examples in the docs includes example output, that the code will provide the correct output.